openid-4-verifiable-presentations-1_0.md

```diff
@@ -215,6 +217,55 @@ Figure: Cross Device Flow
 
 (3) The Wallet prepares the Verifiable Presentation(s) of the Verifiable Credential(s) that the End-User has consented to. It then sends to the Verifier an Authorization Response where the Verifiable Presentation(s) are contained in the `vp_token` parameter.
 
+## Same Device Flow with Wallet Capabilities Discovery {#adv_flow}
```

Comment:
I think {#xxx_xxx} shows in the URL to this section, so I think we should start cleaning it up something like {#capability_discovery_flow}. (this can be a separate issue/PR)

Reply:
I changed the label of this section